product_docs/docs/pgd/5/manual/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Deploying manually
+navigation:
+- 01-provisioning-hosts
+- 02-install-postgres
+- 03-configuring-repositories
+- 04-installing-software
+- 05-creating-cluster
+- 06-check-cluster
+- 07-configure-proxies
+- 08-using-pgd-cli
+---
+
+The best installation process is automatic, which is why PGD is available and deployable using TPA (Trusted Postgres Architect). 
```

Comment:
_Wording Suggestion:_  Automated installation and deployment processes are generally more reliable than manual processes, which is why EDB offers automated PGD deployment using TPA (Trusted Postgres Architect)